src/main/java/com/cisco/trex/stateful/model/stats/PortLatencyData.java

```diff
@@ -0,0 +1,28 @@
+package com.cisco.trex.stateful.model.stats;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class PortLatencyData {
+
+    @JsonProperty("hist")
+    public PortHistData hist;
+
+    @JsonProperty("stats")
+    public PortStatsData stats;
+
+    public PortHistData getHist() {
```

Comment:
it's better to add @JsonProperty(key) annotation for all set/getXXX if it has corresponding field in json response.

Reply:
ok, I will add it.